File: sample_project/android/saas-unisize-sdk-android-java-sample/app/src/main/java/jp/co/makip/saas_unisize_sdk_android_java_sample/MainActivity.java
package jp.co.makip.saas_unisize_sdk_android_java_sample;

import android.Manifest;
import android.content.pm.PackageManager;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.app.ActivityCompat;
import androidx.core.content.ContextCompat;
import androidx.annotation.NonNull;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        enableEdgeToEdge();
        setContentView(R.layout.activity_main);

        // Display the Fragment
        UnisizeFragment fragment = new UnisizeFragment();
        getSupportFragmentManager().beginTransaction()
                .replace(R.id.fragment_container, fragment)
                .commit();
    }

    private void enableEdgeToEdge() {
        // Edge-to-edge feature for Android - should be implemented based on the use case.
    }
}

File: sample_project/android/saas-unisize-sdk-android-kotlin-sample/app/src/main/java/jp/co/makip/saas_unisize_sdk_android_kotlin_sample/MainActivity.kt
package jp.co.makip.saas_unisize_sdk_android_kotlin_sample

import android.Manifest
import android.content.pm.PackageManager
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.app.ActivityCompat
import androidx.core.content.ContextCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        // Fragmentを表示
        val fragment = UnisizeFragment()
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragment_container, fragment)
            .commit()
    }
}
